ui-v2/tests/acceptance/dc/nspaces/index.feature

```diff
@@ -29,5 +29,9 @@ Feature: dc / nspaces / index: Nspaces List
     And I see 1 nspace model with the description "The default namespace"
   Scenario: The default namespace can't be deleted
     Then I see 3 nspace models
+    Then I fill in with yaml
```

Comment:
I added this line to have just the default nspace available to be clicked. Test is still failing.

Reply:
Does looking at the Web Inspector give any clues as to what could cause the test to be failing? Also, considering your comment above re `items` vs `sorted`, does that give any clues as to why it might be failing now?

Reply:
No, no errors or warnings in Web Inspector. This is what the error looks like. 
![Screen Shot 2020-07-31 at 8 55 47 AM](https://user-images.githubusercontent.com/19161242/89036929-a1bf6f00-d30b-11ea-9005-d457984e35c4.png)

The `items` vs `sorted` trick makes me think it might have to do with `uri`/*/*/namespaces`. Not 💯 sure thought 🤔 .

Reply:
Try pausing the test and take a look in the elements panel in Web Inspector also and make sure you are seeing what you are expecting.

The previous test looks like it relied on the `default` namespace being first in the list, which it would be if the `items` in the list aren't `sorted`. If the `items` have any namespaces in them that begin with a, b or c, then `default` will no longer be first in the list, and you'll have to either mock the test out so that `default` is first alphabetically, or just select the position where `default` is when its sorted `And I click on the nspaces.1.actions` to select/click the row at index `1`

Reply:
Yes, that is I added the `search by default` in lines 32-35, to just have that default element display.

Reply:
I added a PR here to help explain a little more, https://github.com/hashicorp/consul/pull/8467 but feel free to just make the amends here if you want and we can close 8467. Let's catch up later it you have any questions around it.